scripts/upgrade

```diff
@@ -94,7 +94,7 @@ ynh_system_user_create --username=$app
 ynh_script_progression --message="Upgrading php-fpm configuration..." --weight=5
 
 # Create a dedicated php-fpm config
-ynh_add_fpm_config
+ynh_add_fpm_config --phpversion=$YNH_PHP_VERSION
```

Comment:
You don't need to provide `--phpversion=$YNH_PHP_VERSION` it's automatically managed: https://github.com/YunoHost/yunohost/blob/1279755463100e6c4a0ba7d4b50f8f39658c8eac/data/helpers.d/php#L86